packages/bpk-component-image/src/BpkBackgroundImage.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+import { BpkSpinner } from 'bpk-component-spinner';
+
+import STYLES from './bpk-background-image.scss';
+
+const getClassName = cssModules(STYLES);
+
+class BpkBackgroundImage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onBackgroundImageLoad = this.onBackgroundImageLoad.bind(this);
+    this.startImageLoad = this.startImageLoad.bind(this);
+
+    this.trackImage = null;
+  }
+
+  componentDidMount() {
+    if (this.props.inView) {
+      this.startImageLoad();
+    }
+  }
+
+  componentWillReceiveProps(newProps) {
+    if (!this.props.inView && newProps.inView) {
+      this.startImageLoad();
+    }
+  }
+
+  onBackgroundImageLoad() {
+    if (this.props.onLoad) {
+      this.props.onLoad();
+    }
+    delete this.trackImage;
+  }
+
+  startImageLoad() {
+    this.trackImage = new Image();
```

Comment:
Yes - this relies on the browser caching the image and therefore only loading the image once